io.github.endless_sky.endless_sky.json

```
@@ -13,6 +13,16 @@
         "--share=ipc",
         "--device=dri"
     ],
+    "add-extensions": {
+        "io.github.endless_sky.endless_sky.plugins": {
+            "directory": "share/games/endless-sky/plugins",
+            "subdirectories": true
+        },
+        "io.github.endless_sky.endless_sky.plugins.endless-sky-high-dpi": {
```

Hyphens are not allowed.

```suggestion
        "io.github.endless_sky.endless_sky.plugins.endless_sky_high_dpi": {
```